File: week6/Week6_day3_ISSS_J/app/src/main/java/com/example/luisenriquez/week6_day3_isss_j/data/remote/RemoteService.java
package com.example.luisenriquez.week6_day3_isss_j.data.remote;

import com.example.luisenriquez.week6_day3_isss_j.data.remote.models.ISSResponse;

import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface RemoteService
{
    @GET("iss-pass.json")
    Single<ISSResponse> getISSResponse(@Query("lat") String lat,
                                       @Query("lon") String lon);
}

File: week6/Week6Day3_ISS/app/src/main/java/com/example/luisenriquez/week6day3_iss/data/remote/RemoteService.kt
package com.example.luisenriquez.week6day3_iss.data.remote.models

import kotlinx.coroutines.experimental.Deferred
import retrofit2.http.GET
import retrofit2.http.Path

interface RemoteService
{
    @GET ("api/location/{WOEID}/{date}")
    fun getWeatherForecast(@Path("WOEID") WOEID:String,
                          @Path("date") date:String)
            : Deferred<List<ISSResponse>>;
}